### DISCUSSION TITLE
Cubase 11 pro (Win) becomes mad when using VST3 (Agridder)

### DISCUSSION THREAD

**Discussion by igbro** [apohl79/audiogridder#894] [Help] [answered]
Hallo Andreas,

Vst2 (audiogridder) plugin host works almost perfectly @ as low as1 buffer (1024 samples in RME 23ms) with my Cubase 11 pro + 2x xeon server (both win10 machines).

But a few Vst3 (agridder) plugin host, makes Cubase become crazy.
From 7/8 instances Cubase starts to reconnect to the server every 2 seconds, giving crazy, huge and changing latencies report on the AudioGridder panel.
This becomes totally unusable.

What I tried with no success to fix this behavior:

Disabling AsioGuard for Agridder
Disabling "Don't process audio int VST plugins when no signal is present"
Increasing latency blocks

At a certain point, OS of the server crashes completely.

At the moment, I really don't mind using only VST2 (works like a charm), but Stienberg just announced that Cubase will drop VST2 support!!!

**Comment by igbro** [comment]
Sorry, forgot to mention 1.2b6

Retrying here, when I reload the VST3 (Agridder) project, no re connections in this case, but Cubase is ASIO overloaded and can't play. Same song, same settings but VST2 (audiogridder) is fluid (with even more instances).

**Comment by JamesXXV** [comment]
I don't know the technical reason for this behavior, but I am going to assume with issues with VST3 itself, the standard not the audiogridder plugin, and audiogridder while it has vst3 support, both the plugin and it being able to run vst3 plugins, vst3 is phenomenally weird, sometimes it won't even give a sound output, it may have some requirements that are different than vst2 (ofc), so I will suggest for the time being do your best to keep using vst2 and hope vst3 becomes better as the days go by, since you have 2x xeon processors I don't think it will be a drag for you to use vst2. vst3 which is said to have lower CPU usage wise among other benefits but real mileage may vary.

**Comment by igbro** [comment]
Thanks for your reply James.

Vst2 are fine and work like a charm at minimal latency with my rig.
My concern and the reason I made those tests with Vst3 is the discontinuity of Vst2 by Steinberg themselves.
Cubase will not support Vst2 anymore.

Vst3 phenomenally weirdness ok, but within its "Mother" DAW?
I got no issue with other Vst3 in Cubase.

This seems to be a latency report/ lan thing witch is obviously not managed the same way by Vst2 (or Juce).
With Vst3, I'm starting well and after 5 or 6 instances of AGridder, Cubase suddenly loses sync, reconnects to Server every 2 seconds or overloads the Asio meter. This is not playable at all I can lead to a crash of the whole server machine system.

This rig can manage 15-20 Audiogridder Vt2's @1 block hosting Acustica Audio heavy plugins. Same project with Vst3's kills it.

**Comment by apohl79** [comment]
VST2 can only do 32bit floats (sample values) while VST3 can do 64bit. If you set Cubase to 64bit processing, AG will also stream audio with that precision over the wire, which doubles the amount of data. Can you try to set Cubase to 32bit and see if that helps (if it is not already the case of course)?

**Reply by daveotero** [reply]
I'm going to try this as well to see if it makes any different on my system. Thanks for the knowledge!

I actually came here to say that I don't have any issues running 1.6b VST3 on my Windows Cubase 11.0.41 system. I rarely run more than 8 instances but have never seen these issues. I know this doesn't help much, but could maybe point to the problem being something specific to your system rather than an inherent issue with Cubase or VST3. Happy to help test further.

**Comment by igbro** [comment]
Hi Andreas.

My Cubase is effectively set to 64bits. This explanation never came to my mind!

I'll try that and will let you know.

Thanks.

**Comment by igbro** [comment, marked as answer]
That was the thing, guys!

It works like vst2 with Cubase engine set to 32bit float!

Thanks.
